subprojects/wrapper-shared/src/main/java/org/gradle/util/internal/PathTraversalChecker.java

```diff
@@ -21,25 +21,27 @@
 
 import static java.lang.String.format;
 
-public class ZipSlip {
+public class PathTraversalChecker {
```

Comment:
This should not be in `:wrapper-shared`. We use this in build cache-related code that has nothing to do with the wrapper.

Reply:
The goal of this PR was not to move the class to a new location but to indicate it has a broader scope than the initial name implied.
Let's move it in a future release. Remember that the class is also used in the wrapper itself, so it is related.